modules/apps/portal-vulcan/portal-vulcan-test-util/src/main/java/com/liferay/portal/vulcan/test/util/HTTPTestUtil.java

```diff
@@ -25,7 +25,7 @@
 import java.nio.charset.StandardCharsets;
 
 /**
- * @author Luis Miguel Barcos
+ * @author Alberto Javier Moreno Lage
```

Comment:
I think Luis Miguel Barcos should also be an author of the class as he was who created it

Reply:
You're right I'm on it